### PR TITLE
Add some docs for in lining included C source as modules

### PR DESCRIPTION
**Pull request by jwerle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -451,3 +451,53 @@ on POSIX systems and a semi-coloon (`;`) on Windows.
 ZZ_MODULE_PATHS="$PWD/path/to/modules:/usr/share/zz/modules" zz build
 ```
 
+#### inline included C source
+
+ZZ supports importing C source with the `using` keyword. Imported C
+source can be inlined at the call size by using the `inline` keyword. In certain
```
Comment (aep) [inline README.md]: Typo s/size/side

File: README.md [hunk 2]
```diff
@@ -451,3 +451,53 @@ on POSIX systems and a semi-coloon (`;`) on Windows.
...
+cases the imported C source may depend on symbols defined in a ZZ
+module. ZZ symbols can be given to an imported C file with the `needs`
+keyword.
+
+The `example_Container` type definition is given to `native.h` and maps
```
Comment (aep) [inline README.md]: Can you try to make that sentence easier to read? The example is good.

**Comment by aep** [comment]
Thanks!


Could you merge all your correction commits into one? It's cleaner when merging to master .


I generally just force push to PR branches, GitHub is handling this correctly

**Comment by jwerle** [comment]
@aep done!

**Comment by aep** [comment]
Thanks!

**Pull request merged**